CV/CV.adoc

```diff
@@ -0,0 +1,94 @@
+image:IMG_20200215_112749.jpg[IMG_20200215_112749,100,60,role=right]
```

Comment:
Good practice for the future: always create a folder named *img* and put all your pictures there